src/app/story/directives/status.component.ts

```diff
@@ -87,6 +91,7 @@ export class StoryStatusComponent implements DoCheck {
     if (this.story) {
       this.setStatus();
       this.isPublished = this.story.publishedAt ? true : false;
+      this.isReleased = this.story.releasedAt ? true : false;
```

Comment:
Nitpick: only show this when not published.  Otherwise, published episodes get this:

![img](http://cav.is/i/0zclf.jpg)

And publishing (you clicked publish, but set a releaseAt in the future) get this:

![img](http://cav.is/i/7sg4t.jpg)

Reply:
good points. fixed in bcbdb44

![screen shot 2018-03-01 at 2 57 53 pm](https://user-images.githubusercontent.com/1205061/36869364-2a659b1c-1d61-11e8-8f7a-251bc28f80c7.png)

![screen shot 2018-03-01 at 2 58 24 pm](https://user-images.githubusercontent.com/1205061/36869369-30ee312e-1d61-11e8-9494-5a76a9d3e37a.png)